### ISSUE TITLE
Towards convergence DMQC methods

### ISSUE THREAD

**Issue by soerenthomsen** [OceanGlidersCommunity/Salinity_SOP#101]
At the moment we list many different implementations of the DMQC corrections.

How could we make progress in converging? This might be an issue when publishing the SOP.

This issue will remain open throughout the whole community review.

For sure we will need some more recommendations under which conditions what approach should be the default choice.

**Comment by soerenthomsen** [comment]
https://www.oceanbestpractices.org writes "A best practice is a methodology that has repeatedly produced superior results relative to other methodologies with the same objective; to be fully elevated to a best practice, a promising method will have been adopted and employed by multiple organizations."

I would like to discuss how we could get here in the long term. i.e. with test datasets and run the code through all different approaches to quantitively show under which environmental condition / platform etc which method gives better results? Until we have not done these method inter-comparisons we will not be able to say.

**Comment by isgiddy** [comment]
It is likely that different methods are suitable in different conditions. But one first way forward would be to create a test dataset that is representative of different regions and run it through all the available methods. 

I think currently, the methods people use are often linked to which platforms they are using, and the code available.

**Comment by soerenthomsen** [comment]
I agree that a test dataset's representing different conditions will be key. This could be done in collaboration with the OceanGliders Datamanagrement Task Team to ensure we use the OG1.0 format etc.
